conf/messages

```diff
@@ -183,10 +183,10 @@ enter-declarant-eori-number.error.required=Enter a valid EORI number
 enter-declarant-eori-number.input-label=Enter the EORI number
 
 #===================================================
-#  ENTER CLAIMANT DETAILS AS INDIVIDUAL PAGE
+#  ENTER CLAIMANT DETAILS AS REGISTERED WITH CDS
 #===================================================
-enter-claimant-details-individual.title=Enter your claimant details as an individual
-enter-claimant-details-individual.help-text=These are the details if we need to contact you about this claim application.
+enter-claimant-details-individual.title=Enter your details as registered with CDS
```

Comment:
could you change this key as well to `enter-claimant-details-as-registered-with-cds`

Reply:
don;t you need to do the same for `enter-claimant-details-importer-company` ?